addons/website/static/tests/tours/create_website.js

```diff
@@ -0,0 +1,84 @@
+
+odoo.define('test_website_modules.tour.test_create_website', function (require) {
+'use strict';
+
+const tour = require('web_tour.tour');
+
+tour.register('test_create_website_1', {
+    test: true,
+    url: '/',
+},
+[
+    {
+        content: "Select the language dropdown",
+        trigger: '.js_language_selector .dropdown-toggle'
+    },
+    {
+        content: "click on Add a language",
+        trigger: 'a.o_add_language',
+    },
+    {
+        content: "Select dropdown",
+        trigger: 'select[name=lang]',
+        run: function () {
+            $('select[name="lang"]').val('"pa_GB"').change();
+        }
+    },
+    {
+        content: "load parseltongue",
+        extra_trigger: '.modal select[name="lang"]:propValueContains(pa_GB)',
+        trigger: '.modal-footer button:first',
+    },
+    {
+        content: "Select the language dropdown",
+        trigger: '.js_language_selector .dropdown-toggle',
+    },
+    {
+        content: "Select parseltongue",
+        trigger: 'a.js_change_lang[data-url_code=pa_GB]',
+    },
+    {
+        content: "Check that we're on parseltongue",
+        trigger: 'html[lang=pa-GB]',
```

Comment:
it is a check, you don't need to click on it.. which is what yo uare doing here, since there is no empty run method